### PR TITLE
Creating src dir for System.IO.IsolatedStorage

### PR DESCRIPTION
**Pull request by dsgouda**
@weshaggard

### CODE REVIEW

**Comment by weshaggard** [comment]
@pallavit please have a look.

File: src/System.IO.IsolatedStorage/src/System.IO.IsolatedStorage.csproj [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <PropertyGroup>
+    <TargetGroup Condition="'$(TargetGroup)'==''">netcore50</TargetGroup>
+  </PropertyGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyName>System.IO.IsolatedStorage</AssemblyName>
+    <OutputType>Library</OutputType>
```
Comment (weshaggard) [inline src/System.IO.IsolatedStorage/src/System.IO.IsolatedStorage.csproj]: Remove OutputType it isn't necessary.

**Comment by weshaggard** [comment]
LGTM other than the one nit.

**Comment by dsgouda** [comment]
Opening new PR at dotnet/master

**Pull request closed without merging**